stubs/grpcio/@tests/test_cases/check_aio.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+import typing
+from typing_extensions import assert_type
+
+import grpc.aio
+
+# Interceptor casts
+client_interceptors = [typing.cast(grpc.aio.ClientInterceptor, "interceptor")]
+grpc.aio.insecure_channel("target", interceptors=client_interceptors)
+
+server_interceptors = [typing.cast(grpc.aio.ServerInterceptor[typing.Any, typing.Any], "interceptor")]
```

Comment:
It's also unusual not to import `Any` directly from typing – at least here in typeshed.

```suggestion
server_interceptors: list[grpc.aio.ServerInterceptor[Any, Any]] = []
```

Reply:
Turns out there's a few places we do that, but I don't really see a good reason. I opened https://github.com/python/typeshed/pull/13761 to standardise.